### DISCUSSION TITLE
add new language

### DISCUSSION THREAD

**Discussion by xiaokai520-corder** [aleksey-hoffman/sigma-file-manager#308] [Q&A]
Hello, I am from the Chinese nation and I would like to add Uyghur language to this software. I can provide a translation for this. Can I add it

**Comment by aleksey-hoffman** [comment]
Hello! Thank you for offering to add the translation
You can find the instructions on this page: [Add new language](https://github.com/aleksey-hoffman/sigma-file-manager/discussions/196). 
Let me know if you need any help with this.
